css/style.scss

```diff
@@ -802,32 +802,39 @@ video {
 
 /* START: move padding fixes and icons-for-tabs capability into core */
 .tabHeaders {
+	display: flex;
 	margin-top: 0;
+	margin-bottom: 10px;
 }
 
 .tabHeaders .tabHeader {
+	flex-basis: 50%;
+	flex-grow: 0;
 	padding: 12px;
-	padding-left: 32px;
-	background-position: 12px;
-	background-repeat: no-repeat;
-	opacity: .5;
-}
-
-.tabHeaders .tabHeader a {
-	color: $color-main-text;
+	text-align: center;
+	border-bottom: 1px solid $color-border;
+	margin-bottom: 0;
+
+	a {
+		padding-left: 32px;
+		background-position: 12px;
+		background-repeat: no-repeat;
+		color: $color-main-text;
+		opacity: .5;
+	}
 }
 
-.tabHeaders .tabHeader.selected,
-.tabHeaders .tabHeader:hover,
-.tabHeaders .tabHeader:focus {
+.tabHeaders .tabHeader.selected a,
+.tabHeaders .tabHeader:hover a,
+.tabHeaders .tabHeader:focus a {
 	opacity: 1;
 }
 
-#tabHeaderChat {
+#tabHeaderChat a {
 	background-image: url('../../../core/img/actions/comment.svg?v=1');
```

Comment:
Btw this breaks when apps are not installed in an app folder in root,
Should fix for 3.1

Reply:
Yep, I only used it because it was already used in the code – by you ;)
https://github.com/nextcloud/spreed/blame/93e0a82f2e91537941c56b1cb31f29e9e4a9338e/css/style.scss#L906

But yes – the reason why I needed to use it like that is because I needed to insert a class (.icon-comment in this case) into the tabHeader, or rather the link inside of it. If we can do that in the code rather, we don’t need that CSS.